back/sssscs/src/main/java/com/ib/user/UserController.java

```diff
@@ -28,9 +22,10 @@ public class UserController {
 	private AuthenticationManager authManager;
 	
 	@PostMapping
-	public ResponseEntity<User> register(@RequestBody UserCreateDto dto) {
-		User user = userService.register(dto);
-		return ResponseEntity.ok(user);
+	public ResponseEntity<User> register(@DTO(UserCreateDto.class) User user) {
+		// FIXME: Return something other than user. Also, why do we return 500 with
+		//        "Failed to write request" now that User implements UserDetails?
+		return ResponseEntity.ok(userService.register(user));
```

Comment:
Didn't return 500 for me?

Reply:
Obsolete comment. https://github.com/magley/sssscs/pull/2/commits/ceec7bb98e542c2181a214165dc4615668b3ce3a